bin/grouper-ctl

```diff
@@ -194,6 +228,15 @@ def main():
     user_parser.set_defaults(func=user_command)
     user_subparser = user_parser.add_subparsers(dest="subcommand")
 
+    user_create_parser = user_subparser.add_parser(
```

Comment:
I'm having trouble getting this help info for sub commands to display. I've tried grouper-ctl --help, grouper-ctl help, grouper-ctl --help sync_db, grouper-ctl help sync_db, grouper-ctl sync_db --help; am I missing the right permutation?